plugin.video.sgtv/resources/search.xml

```diff
@@ -0,0 +1,2 @@
+(lp0
```

Comment:
Thanks. Do you know where the addon userdata folder is? or how to reference it from the addon?